plugins/sgmap.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sgmap
+spec:
+  version: "v0.14.0"
+  homepage: https://github.com/naka-gawa/kubectl-sgmap
+  shortDescription: "A kubectl plugin to visualize security group per pod."
```

Comment:
Delete until V please, it's obvious:)

Reply:
Thanks for your feedback!

I've fixed it in commit c08dbd0. 

Is this what you had in mind?